presto-raptor/src/main/java/com/facebook/presto/raptor/filesystem/HdfsStorageService.java

```diff
@@ -118,7 +121,7 @@ private void createDirectory(Path directory)
     {
         boolean madeDirectory;
         try {
-            FileSystem fileSystem = environment.getFileSystem(FileSystemContext.DEFAULT_RAPTOR_CONTEXT);
```

Comment:
move DEFAULT_RAPTOR_CONTEXT to FileSystemUtil

Reply:
Should I move it as a public or private with getter?